src/drivers/actuators/modalai_esc/modalai_esc.cpp

```diff
@@ -1056,6 +1085,97 @@ void ModalaiEsc::mix_turtle_mode(uint16_t outputs[MAX_ACTUATORS])
 
 }
 
+void ModalaiEsc::handle_actuator_test()
```

Comment:
The built in actuator_test didn't work? https://github.com/PX4/PX4-Autopilot/blob/6991ac014c3fee22ece820b82326a036468e51ee/src/lib/mixer_module/mixer_module.cpp#L433

Reply:
My error, I can just enable outputs if in this test mode and yes it looks like I'm getting values in upateOutputs.  I need to verify mappings, but I'll get rid of this duplication on my side and clean it up.

Reply:
now, I'm simply checking if _actuator_test_sub.updated() and enabling outputs to the ESC if so